examples/run_example.py

```diff
@@ -87,7 +87,9 @@
         print(f"The Average SNR is {avg_snr:.1f}")
  
         # run our standard suite of reconstructions
-        reco_results = reco.do_standard_reco(wave_bundle)
+        reco_results = reco.do_standard_reco(wave_bundle, useful_event.eventNumber)
+
+        pair_idx = reco.get_pair_index(1, 2, reco.pairs_v)
```

Comment:
should the example also show how to get the correlation function? Just for completeness?

Reply:
The correlation functions can't be got outside the class because the getter functions are private.

Reply:
Okay right, I forgot. Nevermind!